nginx/etc/nginx/conf.d/tiler.conf

```diff
@@ -1,25 +1,29 @@
 server {
-    listen 80 default_server;
-    server_name tiles.staging.rasterfoundry.com tiles.rasterfoundry.com;
-    return 301 https://$host$request_uri;
+	listen 80 default_server;
+	server_name tiles.staging.rasterfoundry.com tiles.rasterfoundry.com;
+	return 301 https://$host$request_uri;
 }
 
 upstream tile-server-upstream {
-    server tile-server:9900;
+	server tile-server:9900;
 }
 
 server {
-    listen 443 default_server;
-    server_name tiles.staging.rasterfoundry.com tiles.rasterfoundry.com localhost;
+	listen 443 default_server;
+	server_name tiles.staging.rasterfoundry.com tiles.rasterfoundry.com localhost;
 
-    include /etc/nginx/includes/tiler-security-headers.conf;
+	include /etc/nginx/includes/tiler-security-headers.conf;
 
-    location / {
-        proxy_set_header Host $http_host;
-        proxy_set_header X-Forwarded-For $remote_addr;
-        proxy_read_timeout 20s;
-        proxy_redirect off;
+	location ~^/tiles(.*) {
```

Comment:
I think that it would be good to tweak the block below to something that looks like this:

```diff
diff --git a/nginx/etc/nginx/conf.d/tiler.conf b/nginx/etc/nginx/conf.d/tiler.conf
index a43f0f92..79d55009 100644
--- a/nginx/etc/nginx/conf.d/tiler.conf
+++ b/nginx/etc/nginx/conf.d/tiler.conf
@@ -14,8 +14,8 @@ server {
 
 	include /etc/nginx/includes/tiler-security-headers.conf;
 
-	location ~^/tiles(.*) {
-		return 302 https://$host$1;
+	location ~ /tiles/(.*) {
+		return 302 $scheme://$http_host/$1$is_args$args;
 	}
 
 	location / {
```

Without these changes, I wasn't able to get redirects to work successfully when I configured `TILE_SERVER_LOCATION=http://localhost:9101/tiles`.

I think we could replace `$scheme` and `$http_port` to something that looks more like `https://$host$request_uri`, which would break development, but maintain file consistency (and likely still work in staging and production).

Reply:
Hm. I may have prematurely approved this. I did not test project pages, but if it is breaking there, we should fix it first.

```
return 302 $scheme://$http_host/$1$is_args$args;
```

This was working locally for me, but `https://$host/$1$is_args$args` was not. HTTPS is not enabled locally, so I added `$scheme` to dynamically switch.